packages/experimental/Button/src/useButton.ts

```diff
@@ -19,6 +19,7 @@ export const useButton = (props: ButtonProps): ButtonState => {
       onAccessibilityTap: props.onAccessibilityTap || props.onClick,
       accessibilityLabel: props.accessibilityLabel || props.content,
       accessibilityState: getAccessibilityState(!!disabled || !!loading),
+      enableFocusRing: true,
```

Comment:
Since this is in the default props, we can probably remove it from any macOS specific settings / stylings files? I can do that as a followup if you'd rather not touch that with this PR

Reply:
useButton is only in experimental and i don't see it being set explicitly in any mac files... maybe I'm missing something?

Reply:
.... Thought this was a settings file. My bad.